notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1488 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #89.            problem_type=prediction_type,
</pre><p>Can remove from here, and remove from pipeline inputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>removed</p>